src/CMakeLists.txt

```diff
@@ -129,3 +128,13 @@ target_compile_options(fdbdoc
 
 install(TARGETS fdbdoc RUNTIME DESTINATION bin)
 install(PROGRAMS ${FdbMonitor_EXECUTABLE_PATH} DESTINATION lib/foundationdb/document)
+
+# 60:Don't build packages part of default build
```

Comment:
Just a suggestion - you don't need to add commit message as a comment in the code. People get that information from `git blame`. If there are useful comments for future code reader more than the commit message. Please add them as comments in the code. If you have any comments that are targeted towards the reviewers to make review process easy, please add them as PR comments in the PR, not in the code.